config/runtime.exs

```diff
@@ -229,12 +229,12 @@ case config_env() do
            Enum.join(
              [
                "default-src 'none'",
-               "img-src 'self' cdn.mbta.com #{System.get_env("STATIC_HOST", "")} #{System.get_env("CMS_API_BASE_URL", "")} *.googleapis.com *.gstatic.com *.s3.amazonaws.com data:",
+               "img-src 'self' cdn.mbta.com #{System.get_env("STATIC_HOST", "")} #{System.get_env("CMS_API_BASE_URL", "")} *.google.com *.googleapis.com *.gstatic.com *.s3.amazonaws.com data: i.ytimg.com www.googletagmanager.com",
```

Comment:
question: do we need the `*.s3.amazonaws.com` one? As far as I could tell the only place we need this is for rendering map image tiles on dev (that URL is elsewhere in our config, `mbta-map-tiles-dev.s3.amazonaws.com`).